### PR TITLE
Reformat copy_nest

### PR DESCRIPTION
**Pull request by baturgezici**
copy_nest #2416

### CODE REVIEW

**Review by VedPatwardhan — CHANGES_REQUESTED** [review]
Hi @baturgezici, thanks for contributing, there are a few changes to be made to your PR before we get it merged, please update the `With List input:`, `With Tuple input:` and a `With Dict input` by a single header `With Iterable input`. Secondly please update the examples so that there are no character lists.
Once you're done with the changes, feel free to re-request my review and then we'll look into getting the tests to pass.

Let me know if you have any questions. Thanks 🙂

**Comment by baturgezici** [comment]
Hi @VedPatwardhan, thank you for the review, I've collected all the examples under Iterable and cleared the char list examples. If there is anything I need to update, I look forward to your kind review.

File: ivy/functional/ivy/nest.py [hunk 1]
```diff
@@ -711,6 +711,32 @@ def copy_nest(
     ret
         The copied nest.
 
+    Examples
+    --------
+    With :code:`ivy.Array` input:
+
+    >>> nest = ivy.array([1.,2.,3.],[7.,8.,9.])
```
Comment (VedPatwardhan) [inline ivy/functional/ivy/nest.py:718]: Please update this to `nest = ivy.array([[1.,2.,3.],[7.,8.,9.]])`

**Review by VedPatwardhan — CHANGES_REQUESTED** [review]
Hi @baturgezici, thanks for making the changes, please make this one change which should get the docstring tests passing for your function after which we could merge your PR.
Let me know if you have any questions! Thanks 🙂

File: ivy/functional/ivy/nest.py (hunk 1 above)
Comment (baturgezici) [reply ivy/functional/ivy/nest.py]: Hi, I handled this problem by updating `ivy.array`.

**Review by VedPatwardhan — APPROVED** [review]
lgtm! Thanks for contributing 🙂

**Comment by baturgezici** [comment]
Thanks for your reviews.

**Pull request merged**